composer.json

```diff
@@ -96,7 +98,6 @@
     },
     "conflict": {
         "doctrine/orm": "<2.18",
-        "doctrine/doctrine-bundle": "<2.0 || ^3.0",
```

Comment:
No, you need to keep a conflict here.
It's an optional requirement.
"doctrine/doctrine-bundle": "<2.13",